playlists/excel.yaml

```diff
@@ -9,6 +9,7 @@
   group: Basics
   api_set:
     ExcelApi: 1.1
+  type: public
```

Comment:
Actually, while you're at it, you could also get rid of `host`, since it's also redundant information (since each file is already grouped by host)